sources/postgres/data_test.go

```diff
@@ -280,12 +280,6 @@ func buildConv(spTable ddl.CreateTable, srcTable schema.Table) *internal.Conv {
 	conv := internal.MakeConv()
 	conv.SpSchema[spTable.Name] = spTable
 	conv.SrcSchema[srcTable.Name] = srcTable
-	conv.ToSource[spTable.Name] = internal.NameAndCols{Name: srcTable.Name, Cols: make(map[string]string)}
```

Comment:
Is this not required anymore ?

Reply:
This setup is not needed as its done through code here https://github.com/cloudspannerecosystem/harbourbridge/blob/3e5affad94d0691b260e9f5e8497e7a23147e4d0/internal/mapping.go#L107.